internal/engine/eval/templates/templates.go

```diff
@@ -25,3 +25,9 @@ import (
 //
 //go:embed vulncheckTemplate.tmpl
 var VulncheckTemplate string
+
+// TrustyTemplate is the template for evaluation details of the
+// `trusty` evaluation engine.
+//
+//go:embed trustyTemplate.tmpl
+var TrustyTemplate string
```

Comment:
I would document the accepted parameters as well.

```suggestion
// TrustyTemplate is the template for evaluation details of the
// `trusty` evaluation engine.
//
// This template accepts two parameters, `lowScoringPackages`
// and `maliciousPackages`, which must be list of strings.
//
//go:embed trustyTemplate.tmpl
var TrustyTemplate string
```